FirebaseVertexAI-Docs.podspec

```diff
@@ -0,0 +1,53 @@
+Pod::Spec.new do |s|
+  s.name             = 'FirebaseVertexAI'
+  s.version          = '0.1.0'
+  s.summary          = 'Firebase VertexAI'
+
+  s.description      = <<-DESC
+  Placeholder podspec for docsgen only. Do not use this pod.
+                        DESC
+
+  s.homepage         = 'https://firebase.google.com'
+  s.license          = { :type => 'Apache-2.0', :file => 'LICENSE' }
+  s.authors          = 'Google, Inc.'
+
+  s.source           = {
+    :git => 'https://github.com/firebase/firebase-ios-sdk.git',
+    :tag => 'vertexai-preview-0.1.0'
```

Comment:
I think this should be `'release-10.26'`. Please as a TODO to make this `'CocoaPods-' + s.version.to_s` after release.

Reply:
Done.